cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/mapping/CfOperationsCreator.java

```diff
@@ -25,14 +32,21 @@
      * {@link cloud.foundry.cli.operations}. The cfOperations object
      *
      * @param commandOptions {@link LoginCommandOptions}
+     * @param targetBean     Bean that holds all data that is related to the target space.
```

Comment:
Maybe explicitly allow null values here:
```suggestion
     * @param targetBean     Bean that holds all data that is related to the target space (can be null)
```

Reply:
That's a good idea. For commands which don't accept YAML files, one doesn't need to pass any a bean at all then.